src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let stateClone = { ...state };
+  const result = [];
```

Comment:
result is also a bit of a bad name for variable, smth like `stateClones` will be a great fit